examples/custom_backend/mlkem_native/mlkem/poly.c

```diff
@@ -1 +0,0 @@
-../../../../mlkem/poly.c
```

Comment:
I think this symlink should have been kept?

Reply:
done, must have been a mistake I did during a rebase.

Reply:
This was a bug in the formatter, now fixed (#614)